lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -2,10 +2,24 @@ const log = require("@ui5/logger").getLogger("builder:tasks:bundlers:generateLib
 const moduleBundler = require("../../processors/bundlers/moduleBundler");
 const ReaderCollectionPrioritized = require("@ui5/fs").ReaderCollectionPrioritized;
 
+const DEFAULT_FILE_TYPES = [
+	".js",
+	".control.xml", // XMLComposite
+	".fragment.html",
+	".fragment.json",
+	".fragment.xml",
+	".view.html",
+	".view.json",
+	".view.xml",
+	".properties",
+	".json"
+];
+
 function getDefaultLibraryPreloadFilters(namespace) {
 	return [
 		`${namespace}/`,
 		`!${namespace}/.library`,
+		`!${namespace}/*-preload.js`, // exclude all bundles
```

Comment:
This does not exclude the `designtime/library-preload.designtime.js` or `library-preload.support.js`. Is this intended?

Reply:
But the excludes in line 23 and 24 both will exclude it, don't they?

Reply:
Oh yeah, I missed that. It's friday 😄